build.gradle

```diff
@@ -6,19 +6,19 @@ plugins {
 
 repositories {
     mavenCentral()
-    maven {
-        name "jitpack.io"
-        url "https://jitpack.io"
-    }
-    maven {
-        name "jcenter"
-        url "https://jcenter.bintray.com"
-    }
+    jcenter()
+
+    maven { url "https://jitpack.io" }
+    maven { url "https://jcenter.bintray.com" }
+    maven { url "https://dl.bintray.com/arrow-kt/arrow-kt/" }
 }
 
 dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.70"
     compile "org.bytedeco:llvm-platform:9.0.0-1.5.2"
+    compile "io.arrow-kt:arrow-core:0.10.4"
+    compile "io.arrow-kt:arrow-syntax:0.10.4"
+    compile "io.arrow-kt:arrow-meta:0.10.4"
```

Comment:
This is a lot of dependencies, if we don't end up extracting more from ArrowKt, maybe we could roll our own Option<T> type?